src/Avalonia.Visuals/Rendering/IRenderTimer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Avalonia.Rendering
+{
+    /// <summary>
+    /// Defines the interface implemented by an application render timer.
+    /// </summary>
+    public interface IRenderTimer
+    {
+        /// <summary>
+        /// Raised when the render timer ticks to signal a new frame should be drawn.
+        /// </summary>
+        /// <remarks>
+        /// This event can be raised on any thread; it is the responsibility of the subscriber to
+        /// switch execution to the right thread.
+        /// </remarks>
+        event Action<long> Tick;
```

Comment:
Hmm, should we be producing the `TimeSpan` here? What's the advantage in producing a `long` instead?

Reply:
I'll change it to produce a `TimeSpan`. I only left it producing a `long` since that's what it was when I took over working on the PR.

Reply:
Yeah, I thought that might be the case. I had intended to change it to `TimeSpan` but didn't get to that point I think.